app/helpers/interactive_page_helper.rb

```diff
@@ -1,13 +1,16 @@
 module InteractivePageHelper
   def runnable_activity_page_path(activity, page)
-    path_base = fetch_path_base(activity, @run, page)
-    pass_white_list_params(path_base) if path_base
+    path_base = fetch_path_base(@sequence, activity, @run, page)
```

Comment:
in this case we are using the instance variable `@sequence`, but the other arguments are passed through. The other arguments would also be available as instance variables, so we should be consistent.

Reply:
This is too tricky to improve right now. It seems like the code that deals with pages, activities, and sequences could be improved to be more consistent and easier to understand, but I don't see a clear way to do it yet.